src/LiveComponent/doc/index.rst

```diff
@@ -3733,6 +3733,8 @@ uses Symfony's test client to render and make requests to your components::
             ;
 
             // Submit form data
+	    // 'form' is the name of the form you're submitting (here FormType). 
+            // e.a : form submitted is EntityCreationType the name of your form would be 'entity_creation' and should replace 'form' in the below submitForm call
```

Comment:
```suggestion
            // 'form' is the name of the form you're submitting (here FormType). 
            // e.a : form submitted is EntityCreationType the name of your form would be 'entity_creation' and should replace 'form' in the below submitForm call
```

Reply:
I don't know why i was doing that but when looking at the actual render it was properly aligned... Anyhow, a new commit has been made with a simplification of the comments